exchange-deployer/src/deploy_types/common.rs

```diff
@@ -0,0 +1,27 @@
+multiversx_sc::imports!();
+
+pub const GAS_AFTER_ASYNC: u64 = 10_000;
+
+#[multiversx_sc::module]
+pub trait CommonModule {
+    fn get_default_code_metadata(&self) -> CodeMetadata {
```

Comment:
Receive the address as a parameter and use the new `get_code_metadata()` framework function instead.

Reply:
Nope, no address as parameter, this is just the default value I use.